include/SDL3/SDL_gamepad.h

```diff
@@ -1032,12 +1034,14 @@ extern DECLSPEC int SDLCALL SDL_SendGamepadEffect(SDL_Gamepad *gamepad, const vo
  *
  * \param gamepad a gamepad identifier previously returned by
  *                SDL_OpenGamepad()
+ * \returns 0 on success or a negative error code on failure; call
+ *          SDL_GetError() for more information.
  *
  * \since This function is available since SDL 3.0.0.
  *
  * \sa SDL_OpenGamepad
  */
-extern DECLSPEC void SDLCALL SDL_CloseGamepad(SDL_Gamepad *gamepad);
+extern DECLSPEC int SDLCALL SDL_CloseGamepad(SDL_Gamepad *gamepad);
```

Comment:
The close/destroy functions should probably still return void. What would the application do if it fails?

Reply:
Try the same thing again and hope it works this time?

I agree, this one must have slipped through. I ignored the other destruct functions for this particular reason.

Reply:
Ill reset the haptic and cursor destructors as well. Gimme a few :wink:

Reply:
Done